symbionic/gui/kivy_gui.py

```diff
@@ -148,6 +156,29 @@ def save(self, path, filename):
         self.dismiss_popup()
 
 
+class clsCheckBox:
+    def __init__(self, Logic, cbox_id, number_of_cbs, CB_group):
+        self.Logic = Logic
+        self.cbox_id = cbox_id
+        self.number_of_cbs = number_of_cbs
+        self.CB_group = CB_group
+        self.cbs = []
+        self.ylim = (-150,150)
+
+    def init_cbs(self):
```

Comment:
It's best practice to write out the names (no abbreviations), so that the function behavior is clear.
```suggestion
    def init_check_boxes(self):
```

Reply:
Done!